assets/js/modules/analytics/components/common/ExistingTagNotice.js

```diff
@@ -220,20 +128,12 @@ export default function ExistingTagNotice() {
 		);
 	}
 
-	if ( ua.hasExistingTag ) {
-		if ( ga4.hasExistingTag ) {
-			notice = getNoticeForExistingUAAndGA4Tags();
-		} else {
-			notice = getNoticeForExistingUATag();
-		}
-	} else if ( hasGTMAnalyticsProperty ) {
-		if ( ga4.hasExistingTag ) {
+	if ( hasGTMAnalyticsProperty ) {
```

Comment:
@kuasha420 Since we are only displaying a notice if `hasGTMAnalyticsProperty` is `true`, maybe we can return early if it's `false`.
Then depending on `hasExistingTag`, return the appropriate message.

Reply:
Nice! LGTM!